tests/plugins/pastefromgdocs/manual/pastefromgdocs.md

```diff
@@ -1,4 +1,4 @@
-@bender-tags: feature, 4.13.0, 835, pastetools
```

Comment:
Let's add `target="_blank"` attribute to the links in this test to avoid accidental redirection.

Reply:
I'm not super happy mixing HTML with markdown, but indeed your suggestion makes sense.